src/plugins/intel_cpu/src/emitters/jit_eltwise_emitters.hpp

```diff
@@ -611,5 +610,76 @@ class jit_soft_sign_emitter : public jit_emitter {
+
+    size_t get_inputs_num() const override { return 1; };
+    size_t aux_vecs_count() const override { return host_isa_ == dnnl::impl::cpu::x64::avx512_core ? 0 : 1; }
+
+protected:
+    size_t aux_gprs_count() const override { return (entry_map_.empty() ? 0 : 1) + 1; }
+
+private:
+    void emit_impl(const std::vector<size_t> &inVecIdxs, const std::vector<size_t> &outVecIdxs,
+                   const std::vector<size_t> &poolVecIdxs, const std::vector<size_t> &poolGprIdxs,
+                   const emitter_context *emitContext) const override;
+
+    template <dnnl::impl::cpu::x64::cpu_isa_t isa>
+    void emit_isa(const std::vector<size_t> &inVecIdxs, const std::vector<size_t> &outVecIdxs) const;
+};
+
+class jit_is_inf_emitter : public jit_emitter {
+public:
+    jit_is_inf_emitter(dnnl::impl::cpu::x64::jit_generator *host, dnnl::impl::cpu::x64::cpu_isa_t hostIsa,
+                       InferenceEngine::Precision execPrc = InferenceEngine::Precision::FP32, bool detectNegative = true, bool detectPositive = true)
+            : jit_emitter(host, hostIsa, execPrc), detectNegative(detectNegative), detectPositive(detectPositive) {}
+    jit_is_inf_emitter(dnnl::impl::cpu::x64::jit_generator *host, dnnl::impl::cpu::x64::cpu_isa_t hostIsa, const std::shared_ptr<ov::Node>& node,
+                       InferenceEngine::Precision execPrc = InferenceEngine::Precision::FP32): jit_emitter(host, hostIsa, node, execPrc) {}
+
+    size_t get_inputs_num() const override { return 1; };
+    size_t aux_vecs_count() const override { return host_isa_ == dnnl::impl::cpu::x64::avx512_core || !detectPositive ? 0 : 1; }
+
+protected:
+    size_t aux_gprs_count() const override { return (entry_map_.empty() ? 0 : 1) + 1; }
+
+private:
+    void emit_impl(const std::vector<size_t> &inVecIdxs, const std::vector<size_t> &outVecIdxs,
+                   const std::vector<size_t> &poolVecIdxs, const std::vector<size_t> &poolGprIdxs,
+                   const emitter_context *emitContext) const override;
+
+    template <dnnl::impl::cpu::x64::cpu_isa_t isa>
+    void emit_isa(const std::vector<size_t> &inVecIdxs, const std::vector<size_t> &outVecIdxs) const;
+
+    bool detectNegative;
+    bool detectPositive;
+};
+
+class jit_is_nan_emitter : public jit_emitter {
+public:
+    jit_is_nan_emitter(dnnl::impl::cpu::x64::jit_generator *host, dnnl::impl::cpu::x64::cpu_isa_t hostIsa,
+                       InferenceEngine::Precision execPrc = InferenceEngine::Precision::FP32) : jit_emitter(host, hostIsa, execPrc) {}
+    jit_is_nan_emitter(dnnl::impl::cpu::x64::jit_generator *host, dnnl::impl::cpu::x64::cpu_isa_t hostIsa, const std::shared_ptr<ov::Node>& node,
+                       InferenceEngine::Precision execPrc = InferenceEngine::Precision::FP32) : jit_emitter(host, hostIsa, node, execPrc) {}
+
+    size_t get_inputs_num() const override { return 1; }
+
+protected:
+    size_t aux_gprs_count() const override { return (entry_map_.empty() ? 0 : 1) + 1; }
```

Comment:
Maybe I miss something... But as I can see, we don't use `entry_map_` for these new 3 emitters. Could you clarify please, is it really needed or we can remove this condition?

Reply:
This is just in case the implementation is changed.

Reply:
By the way, I have created the ticket 99765 to cover the same cases to avoid manually addition GPR reg for table